lisa/tools/uptime.py

```diff
@@ -22,12 +21,14 @@ def _check_exists(self) -> bool:
 
     def since_time(self, no_error_log: bool = True, timeout: int = 600) -> datetime:
         # always force run, because it's used to detect if the system is rebooted.
-        command_result = self.run("-s", force_run=True, no_error_log=no_error_log)
-        if command_result.exit_code != 0:
-            raise LisaException(
-                f"get unexpected non-zero exit code {command_result.exit_code} "
-                f"when run {self.command} -s."
-            )
+        command_result = self.run(
+            "-s",
+            force_run=True,
+            no_error_log=no_error_log,
+            expected_exit_code=0,
+            expected_exit_code_failure_message="get unexpected non-zero exit code"
```

Comment:
Can you please check if we need this message? When self.run fails it would log the command and parameters. So this message might be redundant